### DISCUSSION TITLE
Retirer les events en doublon ou remplacer les originaux par ceux-ci

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#1361] [Nouvelles idées]
En listant les events, j'ai remarqué qu'il y a certains en doublon, notamment les events 55, 58, 61 et 62 qui sont les doublons respectifs des events 1, 11, 0 et 9... Du coup, on peut:
- soit remplacer les doublons les moins complets par de nouveaux events (dans ce cas, il faudra en trouver des appropriés sur les zones concernées);
- soit metttre les doublons les plus complets à la place des originaux, ce qui peut certes créer certains bugs, mais optimiser la place pour plus d'events variés;
- soit supprimer les doublons les moins complets.

*Après, si le fait d'avoir des doublons ne choque personne, cette suggestion est à ignorer.*

**Comment by Procion06** [comment]
mettre les doublons les plus complets à la place des originaux : oui ça c'est bien, quitte à les retravailler légèrement
